modules/h2/src/main/java/org/h2/engine/Session.java

```diff
@@ -960,6 +977,11 @@ public void close() {
             try {
                 database.checkPowerOff();
 
+                H2MemoryTracker tracker = memoryTracker;
+                if (tracker != null) {
+                    tracker.close();
```

Comment:
Braces

Reply:
do we apply our code style rules to h2 module too?